app/helpers/workspace_helper.rb

```diff
@@ -29,24 +29,18 @@
 #++
 
 module WorkspaceHelper
+  WORKSPACE_ICON_MAPPING = Hash.new(:project)
+    .with_indifferent_access
+    .merge(
+      portfolio: :briefcase,
+      program: :"project-roadmap"
+    )
```

Comment:
I would rather have a simpler declaration:

```suggestion
  WORKSPACE_ICON_MAPPING = {
      project: :project
      portfolio: :briefcase,
      program: :"project-roadmap"
    ).with_indifferent_access.freeze
```

The current one is hard to read, and iterating over the `WorkspaceHelper::WORKSPACE_ICON_MAPPING`, will not return the `"project" => :project` pair.

```
[26] pry(main)> WorkspaceHelper::WORKSPACE_ICON_MAPPING.each {|k,v| [k, v]}
=> {"portfolio" => :briefcase, "program" => :"project-roadmap"}
```

Reply:
I wanted to keep the logic of defaulting to `:project`, but all usages should not have any problem, so 👍